CONTRIBUTING.md

```diff
@@ -14,17 +14,14 @@ file.
 
 * Download and install the latest stable version of IntelliJ
   - [IntelliJ Downloads](https://www.jetbrains.com/idea/download/)
-  - either the community edition (free) or Ultimate will work.
+  - Either the community edition (free) or Ultimate will work.
 * Start IntelliJ
 * In the Project Structure dialog (`File | Project Structure`), select "Platform Settings > SDKs" click the "+" sign at the top "Add New SDK (Alt+Insert)" to configure an IntelliJ Platform Plugin SDK
-  - point it to the directory of your downloaded IntelliJ Community Edition installation (e.g, `IntelliJ IDEA CE.app/Contents` or `~/idea-IC-183.4886.37`)
-  - change the name to `IntelliJ IDEA Community Edition`
-  - extend it with additional plugin libraries (only until Android Q is released)
```

Comment:
I think we decided that this isn't needed any more because it was for a particular change that eventually was not merged.

Reply:
Actually, added this back because then later it did look like I needed this - So I'm unclear on what's going on here.

Reply:
Yes, you need the plugin SDK to resolve references to IntelliJ classes. Unless you set up a full source configuration, and include all the IntelliJ source files in your project.